index.d.ts

```diff
@@ -1,5 +1,26 @@
-class PolyK {
-    static Slice(p: number[][][], ax: number, ay: number, bx: number, by: number)
+declare var PolyK: {
```

Comment:
This goal to define the Typescript definition isn't suppose to represent a "hacked" class.

The source code for `PolyK` is written as a class with static methods.

```javascript
declare class PolyK {
    static Slice(p: number[], ax: number, ay: number, bx: number, by: number): number[]
    static IsSimple(p: number[]): boolean
    static IsConvex(p: number[]): boolean
    static GetArea(p: number[]): number
    static GetAABB(p: number[]): {}
    static Triangulate(p: number[]): number[]
    static ContainsPoint(p: number[], ax: number, ay: number): boolean
    static Raycast(p: number[],x : number, y: number, dx: number, dy: number): RaycastResult
    static ClosestEdge(p: number[], x: number, y: number): RaycastResult
}
```